src/sqlfluff/rules/L006.py

```diff
@@ -162,7 +167,7 @@ def _eval(self, context: RuleContext) -> EvalResultType:
                         LintResult(
                             anchor=before_anchor,
                             description="Missing whitespace before {}".format(
-                                before_anchor.raw[:10]
+                                before_anchor.raw
```

Comment:
I dunno why this was restricted before and causes incorrect messages to be printed.

For example for `SELECT DATE'2021-02-02'` this is printed:

```
== [test.sql] FAIL                                                                                                                  
L:   1 | P:  12 | L048 | Missing whitespace before '2021-02-0
All Finished 📜 🎉!
```

Reply:
Indeed. But don't chop elsewhere (and probably should create a common function for that if we want to). I say leave as unchopped for now.